lua/lazyvim/plugins/xtras.lua

```diff
@@ -6,6 +6,7 @@ local prios = {
   ["lazyvim.plugins.extras.lang.typescript"] = 5,
   ["lazyvim.plugins.extras.formatting.prettier"] = 10,
   -- default priority is 50
+  ["lazyvim.plugins.extras.overseer.neotest"] = 100,
```

Comment:
Why is this needed?

Reply:
I guess the prioritization of the error makes the error happen, so I write it and push on github to get CI outcome, it's not a right code, I will removed it now.